.changeset/witty-carrots-occur.md

```diff
@@ -0,0 +1,5 @@
+---
+"@saleor/app-sdk": patch
+---
+
+Marked `/middleware`s as deprecated in favor of `/handlers`
```

Comment:
I dont think we even need that, we can write migration guide. Its fine but not necessary